NickvisionMoney.GNOME/Views/AccountView.cs

```diff
@@ -640,10 +640,18 @@ private void ExportToPDF(Gio.SimpleAction sender, EventArgs e)
                     if (dialog.Response == MessageDialogResponse.Suggested)
                     {
                         var newPasswordDialog = new NewPasswordDialog(_parentWindow, _controller.Localizer["PDFPassword"], _controller.Localizer);
-                        password = await newPasswordDialog.RunAsync();
```

Comment:
On line 638, the `async` part of the lambda can be removed since this lambda no longer `await`s anything.